src/main/app/App.js

```diff
@@ -1,21 +1,23 @@
-import React, { Component } from 'react';
-import logo from '../../assets/logo.svg';
+import React from 'react';
+import { App } from 'grommet';
```

Comment:
I would suggest using 'as' syntax and write something like 'GrommetApp' to get rid of _App